spec/operators/findindex-spec.js

```diff
@@ -41,6 +41,35 @@ describe('Observable.prototype.findIndex()', function () {
     expectSubscriptions(source.subscriptions).toBe(subs);
   });
 
+  it('should return index of matching element from source emits multiple elements', function () {
+    var sourceValue = 7;
+    var source = hot('--a--b---c-|', { b: sourceValue });
+    var subs =       '^    !';
+    var expected =   '-----(x|)';
+
+    var predicate = function (value) {
+      return value === sourceValue;
```

Comment:
just minor ergonomics question, what about remove `sourcevalue` and make predicate to compare to value directly as same as https://github.com/ReactiveX/RxJS/pull/1035/files#diff-8e3c11ee811cb3af7dbf4d6817481356R54 does?